config/serverless-operator.yaml

```diff
@@ -71,3 +65,191 @@ repositories:
+      - name: "aws-ovn" # Results in openshift-knative-serverless-operator-<BRANCH>__<OCP_VERSION>-aws-ovn.yaml
+        releaseBuildConfiguration:
+          tests:
+            - as: e2e-aws-ovn-continuous
+              cron: 0 0 * * 0
+              steps:
+                cluster_profile: aws
+                test:
+                  - as: serverless-e2e
+                    commands: make install-mesh test-e2e-with-kafka
+                    # dependencies: Generated by openshift-knative/hack
+                    from: serverless-source-image
+                    resources:
+                      limits:
+                        memory: 6Gi
+                      requests:
+                        cpu: 100m
+                        memory: 200Mi
+                workflow: openshift-e2e-aws-ovn
+      - name: "azure" # Results in openshift-knative-serverless-operator-<BRANCH>__<OCP_VERSION>-azure.yaml
+        releaseBuildConfiguration:
+          tests:
+            - as: e2e-azure-continuous
+              cron: 0 0 * * 0
+              steps:
+                cluster_profile: azure4
+                test:
+                  - as: serverless-e2e
+                    commands: make install-mesh test-e2e-with-kafka
+                    # dependencies: Generated by openshift-knative/hack
+                    from: serverless-source-image
+                    resources:
+                      limits:
+                        memory: 6Gi
+                      requests:
+                        cpu: 100m
+                        memory: 200Mi
+                workflow: openshift-e2e-azure
+      - name: "gcp" # Results in openshift-knative-serverless-operator-<BRANCH>__<OCP_VERSION>-gcp.yaml
+        releaseBuildConfiguration:
+          tests:
+            - as: e2e-gcp-continuous
+              cron: 0 0 * * 0
+              steps:
+                cluster_profile: gcp
+                test:
+                  - as: serverless-e2e
+                    commands: make install-mesh test-e2e-with-kafka
+                    # dependencies: Generated by openshift-knative/hack
+                    from: serverless-source-image
+                    resources:
+                      limits:
+                        memory: 6Gi
+                      requests:
+                        cpu: 100m
+                        memory: 200Mi
+                workflow: openshift-e2e-gcp
+      - name: "hypershift" # Results in openshift-knative-serverless-operator-<BRANCH>__<OCP_VERSION>-hypershift.yaml
+        releaseBuildConfiguration:
+          base_images:
+            hypershift-operator:
+              name: "4.13"
+              namespace: ocp
+              tag: hypershift-operator
+            upi-installer:
+              name: "4.13"
+              namespace: ocp
+              tag: upi-installer
+          releases:
+            latest:
+              integration:
+                include_built_images: true
+                name: "4.13"
```

Comment:
Another place where we have to specify a version and use "integration" stream due to a bug in CI: https://issues.redhat.com/browse/DPTP-3602
Using "integration" instead of "relaase" is a workaround.